e2e/playwright/tests/todo_plugin.spec.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// ***************************************************************
+
+import {expect, test} from '@e2e-support/test_fixture';
+import SlashCommandSuggestions from 'support/components/slash_commands';
+import {fillMessage, getTodoBotDMPageURL} from 'support/utils';
+
+export default {
+    connected: () => {
```

Comment:
This "connected" name is meant to mean "this test is for when the user is connected to the integration"

In the case of the todo plugin, there is no "integration" to connect to, so this is signaling the wrong message. I think we can change this to `slashSuggestions` to be more clear:

```suggestion
    slashSuggestions: () => {
```